src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const styles = sourceString
+    .split(';')
+    .filter(Boolean)
+    .map((style) => style.trim());
+
+  styles.forEach((style) => {
+    const [propery, value] = style.split(':').map((part) => part.trim());
+
+    if (propery && value) {
+      styleObject[propery] = value;
+    }
+  });
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy of the array before the `forEach` operation. The `forEach` method doesn't return a new array and doesn't mutate the original array unless you explicitly modify it inside the callback. You can directly use `forEach` on the result of `map`.